src/sql/azdata.proposed.d.ts

```diff
@@ -1790,6 +1795,77 @@ declare module 'azdata' {
 		 * The path of the parent node.
 		 */
 		parentNodePath: string;
+		/**
+		 * flag to indicate if the node supports filtering
+		 */
+		isFilteringSupported?: boolean;
```

Comment:
Consider removing this and just relying on checking if there's filter definitions defined - it doesn't really make sense to have this be false but have filter definitions on it.